source/cloud-native-platform/health-end-points/index.html.md.erb

```diff
@@ -0,0 +1,30 @@
+---
+title: Health Probes
+last_reviewed_on: 2023-12-12
+review_in: 6 months
+weight: 90
+---
+
+# <%= current_page.data.title %>
+
+## Readiness Checks on Kubernetes
+
+Readiness probes are crucial in Kubernetes as they are called periodically, impacting container availability. 
+Failing readiness checks make containers temporarily unavailable, preventing incoming requests from reaching your application. It's essential to consider what checks to include in readiness probes to avoid unnecessary downtime.
+
+### Considerations:
+
+- Include exclusive/hard dependencies for your service in readiness checks.
+- Avoid including soft dependencies; handle their unavailability in code for a better customer experience.
+
+**Good Example:**
+- A private cache/database (e.g., Redis or Elastic Search) exclusive to the application.
+
+**Bad Example:**
+- Shared components like IDAM, S2S, or CCD.
+
+## Add Database Check to Liveness Probe
+
+A new health check has been added to the liveness probe to monitor database indicators. To enable this on your application, refer to the [example](https://github.com/hmcts/spring-boot-template/blob/master/src/main/resources/application.yaml).
+
+The "Liveness" state of an application indicates whether its internal state allows it to work correctly or recover by itself if failing. A broken "Liveness" state suggests the application cannot recover, prompting infrastructure to restart the application.
```

Comment:
explaining what it is should go before telling someone how to add the database check